### PR TITLE
Commit message

### PR DESCRIPTION
**Pull request by Oheed911**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+pycalc==0.0.2
```
Comment (flrdv) [inline requirements.txt:1]: This repo does not have any versions. What is the reason of this change?

File: .github/workflows/SecurityAnalysis.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+name: Security Analysis
+
+on:
+  push:
+    branches: oheed
+  pull_request:
+    branches: oheed
```
Comment (flrdv) [inline .github/workflows/SecurityAnalysis.yaml:7]: Add also master branch, please

File: .github/workflows/test.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+          pip install -r requirements.txt
+          pip install pytest
+          pip install pylint
+          pip install black
+      - name: Run tests
+        run: pytest tests/testcases.py
+      - name: Run pylint
+        run: pylint std/stdio.py
+      - name: run Black
+        run: black std/stdio.py
```
Comment (flrdv) [inline .github/workflows/test.yaml:29]: Should run them for the whole codebase

**Review by flrdv — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yaml (hunk 3 above)
Comment (Oheed911) [reply .github/workflows/test.yaml:29]: ok, I am working on adding this for whole project.

File: .github/workflows/SecurityAnalysis.yaml (hunk 2 above)
Comment (Oheed911) [reply .github/workflows/SecurityAnalysis.yaml:7]: ok I will update.
